### PR TITLE
Add value for istio specifying k3d as deployment

### PR DESCRIPTION
**Pull request by barchw**
**Description**

Changes proposed in this pull request:

- Add k3d parameter used when deploying Kyma on k3d

**Related issues**

- https://github.com/kyma-project/cli/issues/1373

### CODE REVIEW

**Review by clebs — DISMISSED** [review]
This does not need a flag. The CLI is capable of detecting if a cluster is k3d.
Please see:
https://github.com/kyma-project/cli/blob/7a600e8151b5fd23ff1be0f79c31d8259999714d/internal/clusterinfo/k3d.go#L17

File: internal/deploy/values/values.go [hunk 1]
```diff
@@ -59,7 +59,8 @@ func addClusterSpecificDefaults(builder *builder, clusterInfo clusterinfo.Info)
 			addDefaultServerlessRegistryConfig(defaultRegistryConfig).
 			addDefaultGlobalDomainName(defaultLocalKymaDomain).
 			addDefaultGlobalTLSCrtAndKey(defaultLocalTLSCrtEnc, defaultLocalTLSKeyEnc).
-			addDefaultServerlessKanikoForce()
+			addDefaultServerlessKanikoForce().
+			addDefaultk3dValuesForIstio()
```
Comment (clebs) [inline internal/deploy/values/values.go:63]: This should be added only if the target cluster is K3d.

**Review by clebs — CHANGES_REQUESTED** [review]

File: internal/deploy/values/values.go (hunk 1 above)
Comment (barchw) [reply internal/deploy/values/values.go:63]:
```go
if k3d, isK3d := clusterInfo.(clusterinfo.K3d); isK3d {
```
This is not the correct place to have this param added?

Comment (clebs) [reply internal/deploy/values/values.go:63]: Ah yes, sorry... I missed the fact that this call is inside a block that only runs if the cluster is k3d.

**Review by clebs — APPROVED** [review]

**Pull request merged**